### PR TITLE
feat: minor improvements

### PR DESCRIPTION
**Pull request by FionnD**
Some minor improvements from other PRs

### CODE REVIEW

**Comment by FionnD** [comment]
hi @aga11313 @alexcg1 @theUnkownName 

Do we need both the table and bullet points in the overview section?

**Comment by aga11313** [comment]
> hi @aga11313 @alexcg1 @theUnkownName 
> 
> Do we need both the table and bullet points in the overview section?
> 
> Screenshot 2021-05-12 at 16 54 17
> 

Hm the table is more very specific info? And the prose section is more like a story telling information. I would keep them both personally just keep the second part more brief

File: example_template.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 # Run the EXAMPLE NAME
+*You can also include a gif with a full demo of the example*
+
+
+ *ADD A TABLE OF CONTENTS HERE *
```
Comment (alexcg1) [inline example_template.md:5]: I think examples already used an automated TOC creator, no? I think it was part of CI/CD. Just need to put the right tags in the code.

I nabbed this from our ancient South Park example:

```markdown
<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
**Table of Contents**

- [Overview](#overview)
- [Try out this demo by yourself](#try-out-this-demo-by-yourself)
- [Prerequirements](#prerequirements)
- [Prepare the data](#prepare-the-data)
- [Define the Flows](#define-the-flows)
- [Run the Flows](#run-the-flows)
- [Dive into the Pods](#dive-into-the-pods)
- [Wrap up](#wrap-up)
- [Next Steps](#next-steps)
- [Documentation](#documentation)
- [Stay tuned](#stay-tuned)
- [License](#license)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```

Stuff between tags is auto-generated each time. I assume it should still work for current examples

Comment (FionnD) [reply example_template.md:5]: I always assumed you had to run doctoc locally? I don't see it mentioned anywhere in our github workflow?

**Comment by theUnkownName** [comment]
> I would like to keep it short, and prevent text heavy. But I'm not to pushed about it.
> 
> > hi @aga11313 @alexcg1 @theUnkownName
> > Do we need both the table and bullet points in the overview section?
> > Screenshot 2021-05-12 at 16 54 17
> 
> Hm the table is more very specific info? And the prose section is more like a story telling information. I would keep them both personally just keep the second part more brief

I agree with Agnes. I would vote to keep only the table. It explains it better and we shouldn't be so verbose with the bullet points.

**Comment by FionnD** [comment]
> > I would like to keep it short, and prevent text heavy. But I'm not to pushed about it.
> > > hi @aga11313 @alexcg1 @theUnkownName
> > > Do we need both the table and bullet points in the overview section?
> > > Screenshot 2021-05-12 at 16 54 17
> > 
> > 
> > Hm the table is more very specific info? And the prose section is more like a story telling information. I would keep them both personally just keep the second part more brief
> 
> I agree with Agnes. I would vote to keep only the table. It explains it better and we shouldn't be so verbose with the bullet points.

So only the table?

File: example_template.md (hunk 1 above)
Comment (alexcg1) [reply example_template.md:5]: Ah, maybe? I guess I didn't often change the headings when I edited examples so the issue never arose for me.

Want to run a quick test? Try replacing your TOC with the empty doctoc tags and see what happens? Can always revert. if it works, saves a lot of time of time for everyone in future. If not, just revert and it's like 2 minutes of wasted effort

**Comment by theUnkownName** [comment]
I would say only the table and not the bullet points, yes

File: example_template.md (hunk 1 above)
Comment (FionnD) [reply example_template.md:5]: Tested and it didn't work! You have to run it locally. So I would leave it in the template and require people to manually make sure it works.

**Comment by FionnD** [comment]
Update: 
1. I've add a TOC to remind people to add it. 
2. I've remove the paragraph under the table to streamline the template.

**Review by theUnkownName — APPROVED** [review]

**Pull request merged**